New topic Insights & analytics (#15373)
docker/docker.github.io: docker-hub/publish/customer_faq.md

keywords: Docker, docker, store, purchase images
title: Docker Hub Publisher & Certified Content FAQs
redirect_from:
  - /docker-store/customer_faq/
---

## Hub publisher profile

### What is the difference between a Community User and a Verified Publisher?

> Community users are users who have signed up to receive a Docker ID and made some of their repositories public on Docker Hub. By contrast, Verified Publishers have engaged directly with Docker's partner team, and enrolled in our Docker Technology Partner program. Verified Publisher profiles indicate that the business entity of the publisher has been validated, and have taken proactive steps to partner with Docker. Unlike community content, Verified Publisher profiles indicate true provenance of the content.

Reviewer: I'm not sure this page is actually visible any more? It references a load of old store functionality, and I can't navigate to it on docs.docker.com. Maybe it redirects elsewhere and we can delete it?

Reviewer 2: yeah I wasn't sure either but it looks like this and a bunch of other pages in this directory never get published, and are all redirected to the index page. Let's leave these pages hidden as they are now, and I will open a new PR to clean up and remove unused files later.

Reviewer: 👍

### What is the difference between a Community User and a Verified Publisher?

> Community Users have simply signed up to receive a DockerID and made some of their repositories Public on Docker Hub. By contrast, Verified Publishers have engaged directly with Docker's partner team, and enrolled in our Docker Technology Partner program. These profiles indicate that the business entity of the publisher has been validated, and have taken proactive steps to partner with Docker. Unlike Community content, these Verified Publisher profiles indicate true provenance of the content.

Reviewer 2: ```suggestion
Community users are users who have signed up to receive a Docker ID and made some of their repositories public on Docker Hub. By contrast, Verified Publishers have engaged directly with Docker's partner team, and enrolled in our Docker Technology Partner program. These profiles indicate that the business entity of the publisher has been validated, and have taken proactive steps to partner with Docker. Unlike community content, these Verified Publisher profiles indicate true provenance of the content.
```
